### ISSUE TITLE
Link text for "Help" link is not read by Talkback

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#688]
Scenario to reproduce

Samsung Galaxy J3 Prime
Android version 7.0

Samsung S5
Android version 6

App version: 1.0 (48)

While Talkback is on, it just announces "opens in a new window" link for the "Help" link not the link text "Help".

Expected result
Talkback should announce the link text.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html

Screen
![link](https://user-images.githubusercontent.com/12256067/87466513-5fe0ba00-c5e4-11ea-833c-ae4746d0a8af.gif)
shots
